admyral/workers/action_executor.py

```diff
@@ -79,7 +81,11 @@ def sync_execute(
                 # make available to the function as a contextvar
                 ctx.set(exec_ctx)
 
-                result = func(**args)
+                # check for wait node => don't call the function
```

Comment:
Explanation comment could be better:

```suggestion
                # for wait actions, we skip the function call because the waiting is already executed in the WorkflowExecutor
```

Please also add the comment to the async action handling.